05-pages/landing-pages/landing-pages.stories.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import Layout from '../../04-templates/Layout/Layout.component';
+import { items as menu } from '../../02-molecules/menus/MainMenu/MainMenu.data.json';
+import { items as footerItems } from '../../02-molecules/menus/InlineMenu/InlineMenu.data.json';
+
+import Hero from '../../02-molecules/Hero/Hero.component';
+import Heading from '../../01-atoms/text/Heading.component';
+import TextWithImage from '../../02-molecules/TextWithImage/TextWithImage.component';
+import Hr from '../../01-atoms/hr/Hr.component';
+import Grid from '../../03-organisms/grid/Grid.component';
+import NewsCard from '../../02-molecules/cards/NewsCard/NewsCard.component';
+import Cta from '../../02-molecules/cta/Cta.component';
+
+/**
+ * Storybook Definition.
+ */
+export default { title: 'Pages/Landing Pages/Home' };
```

Comment:
We can probably remove the `Home` here so `Home` isn't duplicated in the storybook sidebar nav. 
![Screenshot-2020-06-16-09-52-57-210x194](https://user-images.githubusercontent.com/366413/84790628-2e5dda00-afb7-11ea-8e7f-225b41c314e9.png)

Reply:
done